mcp-server-open-api-spec/README.md

```diff
@@ -27,21 +27,26 @@ To use this template, you need:
 2. Credentials for your REST API; the full set of supported authentication types (OAuth Machine to Machine, OAuth User to Machine, Bearer token, etc is described [here](https://docs.databricks.com/aws/en/query-federation/http#authentication-methods-for-external-services)
 3. An installation of the [Databricks CLI](https://docs.databricks.com/aws/en/dev-tools/cli/install)
 
-### Clone the app code
+### 1. Fetch app code
```

Comment:
Should we get rid of the numbers? I don't know if they add much value + they seem easy to get wrong